requirements.txt

```diff
@@ -0,0 +1,43 @@
+opentelemetry-distro==0.28b1
+opentelemetry-instrumentation==0.28b1
+opentelemetry-exporter-otlp-proto-http==1.9.1
+opentelemetry-propagator-jaeger==1.9.1
+opentelemetry-propagator-b3==1.9.1
+opentelemetry-propagator-aws-xray==1.0.1
+opentelemetry-propagator-ot-trace==0.28b1
+opentelemetry-instrumentation-aws-lambda==0.28b1
+opentelemetry-instrumentation-dbapi==0.28b1
+opentelemetry-instrumentation-logging==0.28b1
+opentelemetry-instrumentation-sqlite3==0.28b1
+opentelemetry-instrumentation-urllib==0.28b1
+opentelemetry-instrumentation-wsgi==0.28b1
```

Comment:
Any reason not to include these as optional dependencies? Seems a bit much to force the user to install all this stuff, most of which they won't need.

Reply:
This depends on whether you mostly expect users to consume this repo as a package they include in their project, or as a standalone binary they install separately. In the latter case, this doesn't really matter, but we should also consider supplying an installation method more self-contained than `pip install`. And in the former case, it can make a big difference, depending on what all of these packages pull in as their own dependencies.

Reply:
Right, we've checked the size of the instrumented packages, now instrumented packages are about 5MBs.

Reply:
How did you calculate this? By checking the size of `site-packages` before and after install?

Reply:
Installed instrmented packages in venv, checked size before and after.